### PR TITLE
pylint improvements

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
we are close - really close ;-)

### CODE REVIEW

File: update-bot/wlmbots/lib/template_checker.py [hunk 1]
```diff
@@ -11,6 +11,7 @@ def __init__(self, config=None):
             config = {}
         self.tpl_match_regex = None
         self.config = config
+        self._config = config
```
Comment (gbirke) [inline update-bot/wlmbots/lib/template_checker.py]: Did pylint complain here? Was the complaint that the internal field self._config was not initialized in the constructor? That double assignment looks weird to me. Maybe you can change it to 
```python
self._config = None
self.config = config
```

Comment (WMDE-Fisch) [inline update-bot/wlmbots/lib/template_checker.py]: yes he complains when an field is not initialized in the constructor :-)

Comment (wmde-manicki) [inline update-bot/wlmbots/lib/template_checker.py]: Sorry if I will say a complete rubbish here (and sorry that it does not directly relate to pylint checks) but how do these two (I mean `self.config` and `self._config`) actually differ? I have only found `self._config` being set in `config` setter and returned in `config` getter. Is this used for something else? If so, how about some other name for one (or both) of these two fields?

Another thing: how about changing the default value of `config` parameter of the constructor to `{}`? This seems to be the actual default value, not a `None`.

Comment (WMDE-Fisch) [inline update-bot/wlmbots/lib/template_checker.py]: Good question - no rubbish - and {} sounds as well better to me. I guess _config is used somwhere in pywikibot? @gbirke

**Pull request merged**